composer/packages/theme/scripts/build-theme.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
copyright year 2019

Reply:
Actually this was created last year (2018) and forgot to update. Yes need to change it since we are merging only now. I'll fix it. Thanks 👍

Reply:
Done.